samples/getting-started/simulation/LargeSimulation.ipynb

```diff
@@ -0,0 +1,2459 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "acquired-exhibit",
+   "metadata": {},
+   "source": [
+    "# Simulating quantum programs with large number of qubits\n",
+    "\n",
+    "The quantum programs with large number of qubits can be efficiently simulated using the [sparse simulator](https://docs.microsoft.com/azure/quantum/user-guide/machines/sparse-simulator).\n",
+    "\n",
+    "The sparse simulator is a simulator that utilizes a sparse representation of quantum state vectors, as opposed to the [full-state simulator](https://docs.microsoft.com/azure/quantum/user-guide/machines/full-state-simulator). This feature allows the sparse simulator to minimize the memory footprint used to represent quantum states, thus enabling simulations over a larger number of qubits. The sparse simulator is efficient for representing quantum states that are sparse in the computational basis, that is, quantum states for which most of the amplitude coefficients are zero in the computational basis. As such, sparse simulator enables users to explore larger applications than what can be represented using the full-state simulator which will waste both memory and time on an exponentially large number of zero-amplitudes.\n",
+    "\n",
+    "For more information about the sparse simulator, please see [Jaques and Häner (arXiv:2105.01533)](https://arxiv.org/abs/2105.01533).\n",
+    "\n",
+    "In this sample, we'll look at the advantages of the sparse simulator over the full-state simulator.\n",
+    "\n",
+    "Below, we'll create a few clusters (groups) of qubits, each prepared in a [GHZ state](https://en.wikipedia.org/wiki/Greenberger%E2%80%93Horne%E2%80%93Zeilinger_state)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "prime-horror",
+   "metadata": {},
+   "source": [
+    "To run the cells below, click the cell and press \\<Ctrl+Enter>. The same shortcut will let you exit from the Edit mode of a Markdown cell like the one you are reading. To see the other shortcuts press \\<Esc>, \\<h>. Press \\<Ctrl+Shift+p> to see the command pallete. \\<Esc> to exit."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "shaped-office",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "application/x-qsharp-data": "[]",
+      "text/html": [
+       "<ul></ul>"
+      ],
+      "text/plain": []
+     },
+     "execution_count": 1,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "open Microsoft.Quantum.Arrays;        // Contains functions for creating and manipulating arrays of data.\n",
```

Comment:
This may be hard to read on narrower screens; I'd suggest moving the comments out to a text cell before this code cell. Perhaps something along these lines?

> Below, we'll open a few Q# namespaces to provide access to functions for creating manipulating arrays of data, and for helping to visualize our quantum programs. To learn more, check out the [sample on visualizing quantum programs](https://github.com/microsoft/Quantum/tree/main/samples/diagnostics/visualization) and the [Q# API reference](https://docs.microsoft.com/qsharp/api/qsharp/).